README.md

```diff
@@ -23,40 +23,42 @@ These steps assume your OpenShift deployment has the default set of ImageStreams
 
 		$ oc new-app openshift/templates/cakephp.json -p SOURCE_REPOSITORY_URL=<your repository location>
 
-4. Watch your build progress  
+4. Depending on the state of your system, and whether additional items need to be downloaded, it may take around a minute for your build to be started automatically.  If you do not want to wait, run
+
+   	     	$ oc start-build cakephp-example
```

Comment:
this is rendering with an extra indentation inside the code block (probably an issue in your PR for the other repos too).  guessing it's a tabs vs spaces issue.  use all spaces or two tabs.